### PR TITLE
Plugin manager using attributes for plugins

### PR DESCRIPTION
**Pull request by chesn0k**

### CODE REVIEW

File: src/Command/PluginManager.php [hunk 1]
```diff
@@ -36,6 +36,7 @@ protected function generate(array &$vars, Assets $assets): void {
 
     $discovery_types = [
       'annotation' => 'Annotation',
+      'attribute' => 'Attribute',
```
Comment (chesn0k) [inline src/Command/PluginManager.php:39]: Should hide this option for Drupal 10.1?

Comment (Chi-teck) [reply src/Command/PluginManager.php:39]: I think no. D10.1 will have EOL this year, and besides seems this discovery is not breaking existing 10.1 sites.

**Comment by Chi-teck** [comment]
Looks great. Thank you.

New release will be published once we fix existing tests. This is currently blocked by Composer incompatibility with Symfony 7. See https://github.com/composer/composer/issues/11736

**Comment by andypost** [comment]
The PR missing to add **deriver** property to attribute which should exists in every plugin

**Comment by chesn0k** [comment]
@Chi-teck do need new PR for this?

**Comment by Chi-teck** [comment]
@chesn0k that's already fixed. See #127.

**Pull request merged**